PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -37,7 +37,9 @@ public static class MemoryManagement
 		public static bool CheckMemory(bool minor, string eventDescription, bool okToBotherUser)
 		{
 			var heapMem = GC.GetTotalMemory(true); // first, as it may reduce other numbers
+			var is64BitProcess = IntPtr.Size == 8; // according to MSDN
 			long memorySize64;
```

Comment:
The trailing 64 for property names in properties of Process makes them longs; there are (deprecated) properties with the shorter names that return ints. I think it's at least somewhat helpful for a variable that is mainly intended to hold the result of something like proc.memorySize64 to be called memorySize64. But could go either way.